File: Selenium/Activities/Activity4.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity4 {
	
public static void main(String[]args) {
		
		WebDriver driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net/selenium/target-practice");
		driver.getTitle();
		System.out.println("Home page title: " + driver.getTitle());
		
		String thirdheaderText = driver.findElement(By.xpath("//h3[@id='third-header']")).getText();
		System.out.println(thirdheaderText);
		
		String fifthHeaderColor = driver.findElement(By.xpath("//h5[text()='Fifth header']")).getCssValue("color");
        System.out.println(fifthHeaderColor);
        
        String violetButtonClass = driver.findElement(By.xpath("//button[text()='Violet']")).getAttribute("class");
        System.out.println(violetButtonClass);
		
        String greyButtonText = driver.findElement(By.xpath("//button[text()='Grey']")).getText();
        System.out.println(greyButtonText);
		
		driver.close();
		
	}

}

File: Selenium/Activities/activity4.py
from selenium import webdriver
from selenium.webdriver.firefox.service import service as FirefoxService
from selenium.webdriver.common.by import By
import time

with webdriver.Firefox() as driver:
    #Open URL
    driver.get("https://v1.training-support.net/selenium/target-practice")
    print("Page Opened: ", driver.title)

    third_heading = driver.find_element(By.XPATH, "//h3[@id='third-header']")
    print("Header text: ", third_heading.text)

    fifth_heading = driver.find_element(By.XPATH, "//h5[@class='ui green header']")
    print("Header color: ", fifth_heading.value_of_css_property("color"))

    violet_button = driver.find_element(By.XPATH, "//button[@class='ui violet button']")
    print("Violet Button class: ", violet_button.get_attribute("class"))

    grey_button = driver.find_element(By.CLASS_NAME, "grey")
    print("Grey button's text is: ", grey_button.text)



                        